Updating README with new format

1Copenut opened the pull request:
Worked with Gretchen to define the new format and close out the [older PR for ROEs](https://github.com/department-of-veterans-affairs/va.gov-team/pull/5881)

1Copenut committed:
Updating README with new format

Worked with Gretchen to define the new format and close out the [older PR for ROEs](https://github.com/department-of-veterans-affairs/va.gov-team/pull/5881)

jenstrickland commented on platform/accessibility/README.md:
```diff
@@ -1,24 +1,77 @@
-* Learn about [accessibility best practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md) in design and engineering
-* Learn how to request a [508 pre-launch review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-request-prelaunch-review.md) for your product or feature improvement
+VSP Accessibility supports teams building on VA.gov to ensure that nobody is excluded from using products on VA.gov. We provide best practice guidance throughout the development cycle, build and recommend automated testing tools to flag accessibility issues, reducing the manual effort required to fix issues immediately before launch and post-la
+
```
Looks like content is missing here.

1Copenut replied:
It has been moved further down the page, in our Resources and Guides section.

jenstrickland commented on platform/accessibility/README.md:
It might be easier to see if the relevant resources were inset list items, like the following bullets' items.

jenstrickland commented on platform/accessibility/README.md:
considerations for planning to include people with disabilities in user research

jenstrickland commented on platform/accessibility/README.md:
I think providing guidance on possible design system components and patterns would be in the design intent collaboration point. By the Discover phase, designers will have already started design, since we only get to speak with them in Usability Testing Prep, when they are expected to have designs ready.

1Copenut replied:
I'll add a bit about the design system when we have governance better defined

jenstrickland commented on platform/accessibility/README.md:
As mentioned in the Define stage, considerations for planning to include diverse research would be helpful earlier, as recruiting people with disabilities can take quite a bit of time, as can planning research.

jenstrickland commented on platform/accessibility/README.md:
I think this testing scenarios would also be in the design intent (Define) stage. 

In the Discover phase, we only have Usability Testing Prep involvement, and that would be evaluating designs and research plans.

1Copenut replied:
Define feels too early to be scoping test scenarios. I think I'll leave it here for now, and adapt if we see that being a better fit later.

jenstrickland commented on platform/accessibility/README.md:
Would it be possible to provide this info as a bulleted list? 

In addition to the engineering-related content provided here, we also evaluate the interactions and design during a Staging Review.

jenstrickland commented on platform/accessibility/README.md:
Is it possible to compose this into a bulleted list?

jenstrickland commented on platform/accessibility/README.md:
Perhaps Inclusive User Stories?  I know I had it as user stories, but reading it now it isn't as clear these are specific to a11y / inclusive design.

jenstrickland reviewed (changes requested):
I provided single comments throughout.

meganhkelley commented:
Thanks for working on this, @1Copenut ! Assuming you've worked with @gmaciolek on it and she has reviewed, I don't need to review this. ✅ 👍

1Copenut committed:
Update README.md

Updating based on feedback from VSA accessibility specialist, and VSP teammates
